### ISSUE TITLE
Improving/changing of the `attrs-newline` rule

### ISSUE THREAD

**Issue by MorevM** [yeonjuan/html-eslint#303]
Good day :)
Thank you for creating and supporting this.

Based on the [documentation of the rule](https://html-eslint.org/docs/rules/attrs-newline), I was actually expecting a different behavior - that if the attributes count in one row is greater than `ifAttrsMoreThan`, then a linebreaks are inserted.
The same behavior as realized in [`vue/max-attributes-per-line`](https://eslint.vuejs.org/rules/max-attributes-per-line) rule.

It came as a surprise to me that in case I want to write attributes on different lines and the number of attributes is less than specified in the config, the reverse validation happens:

```js
'@html-eslint/attrs-newline': ['warn', {
  closeStyle: 'newline',
  ifAttrsMoreThan: 3,
}]
```

```html
<!-- This will be reported -->
<button
  type="button"
  class="app-button app-button--variant-secondary app-button--block app-button--wide"
><!-- --></button>

<!-- And autofixed to this: -->
<button type="button" class="app-button app-button--variant-secondary app-button--block app-button--wide">
  <!-- -->
</button>
```

This is actually quite inconvenient, I'd like the rule to only trigger if too many attributes are listed in one line, while still leaving the choice for the developer to write fewer attributes in any format.

It seems to me that this should be the default behavior (I can think of many more examples where the current implementation would get in the way - tell me if you need to), but if you don't want a breaking change, you could make the change as an extra option.

I can do it myself, if we agree on what it needs to be done and how the API for configuration should look like :)

**Comment by yeonjuan** [comment]
Hi, @MorevM Thank you for your report.
> This is actually quite inconvenient, I'd like the rule to only trigger if too many attributes are listed in one line, while still leaving the choice for the developer to write fewer attributes in any format.

 I agree to change the behavior of “ifAttrsMoreThan” as you suggested. 👍
